### DISCUSSION TITLE
Curso Fundamentos, editor de texto.

### DISCUSSION THREAD

**Discussion by GustavEvangelista** [balta-io/forum#373] [Tira Dúvidas] [answered]
Estou com uma dificuldade na hora de salvar o arquivo no editor de texto. Quando eu digito o caminho para salvar aparece um erro, qualquer que seja o caminho que digito. 

![image](https://github.com/balta-io/forum/assets/141782960/7911aa33-d416-4315-8cff-e0d630a2e797)

o que pode ser? não parece ser erro no código.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @GustaD3V, em qual caminho está tentando salvar? Se o usuário não tiver permissão não rola...

**Reply by GustavEvangelista** [reply]
Olá André, eu sou realmente leigo no assunto, como faço pra saber se tenho ou não permissão? O computador é  pessoal, pq eu precisaria de permissão para salvar no meu próprio computador?  é alguma configuração no VScode?

estou tentando salvar nesse caminho: 

C:\Users\gusta\OneDrive\Escritorio\C#

mas todas as tentativas com caminhos diferentes deram o mesmo erro, mesmo executando o VScode como administrador.

**Reply by GustavEvangelista** [reply]
Balta, perdão, já entendi onde eu estava errando, tenho que informar que é um arquivo txt usando o .txt no nome. Pode fechar o post. Agradeço o retorno. Que vergonha. kkk

**Reply by andrebaltieri** [reply]
Show, imagina! Não tem dúvida besta!
